looprpc/client.proto

```diff
@@ -710,6 +710,41 @@ message LiquidityParameters{
     A set of liquidity rules that describe the desired liquidity balance.
     */
     repeated LiquidityRule rules = 1;
+
+    /*
+    The maximum fee paid to the server for facilitating the swap, expressed
+    as parts per million of the swap volume.
+    */
+    uint64 max_swap_fee_ppm = 2;
+
+    /*
+    The maximum fee paid to route the swap prepay and invoice, expressed as
+    parts per million of the volume being routed.
+    */
+    uint64 max_routing_fee_ppm = 3;
```

Comment:
Isn't it that this value is only matched against a hard-coded constant now? Perhaps it is better to use `max_routing_fee_ppm` as the fee limit directly.